docs/src/main/asciidoc/firestore.adoc

```diff
@@ -0,0 +1,60 @@
+== Cloud Firestore
+
+Spring Cloud GCP provides convenience starter which automatically configures authentication settings and client objects needed to begin using the https://cloud.google.com/firestore/[Google Cloud Firestore].
```

Comment:
I think you need specify somewhere that this is for Firestore in **Native mode**.